.github/workflows/auto_pr.yml

```diff
@@ -0,0 +1,82 @@
+# Make sure you have the secrets MACHINE_USER_PAT and MACHINE_USER_FORKED_REPOSITORY set
+
+name: Create official images PR
+
+on:
+  push:
+    branches:
+      - master
+
+    paths:
+      - "generate-stackbrew-*.sh"
+      - "**/Dockerfile"
+      - "**/architectures"
+
+jobs:
+  auto-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the docker-node repo
+        uses: actions/checkout@v2
+        with:
+          path: docker-node
+
+      - name: Checkout the official-images repo
+        uses: actions/checkout@v2
+        with:
+          path: official-images
+          repository: docker-library/official-images
+
+      - name: Generate PR text
+        uses: actions/github-script@v3
+        id: pr-format
+        with:
+          script: |
+            const basePath = `${process.env.GITHUB_WORKSPACE}/docker-node`;
+            const script = require(`${basePath}/.github/prFormat.js`);
+            return script({ github, context, maintainersFile: `${basePath}/ACTIVE_MAINTAINERS` });
+
+      - name: Generate Stackbrew for diff
+        run: |
+          cd docker-node
+          ./generate-stackbrew-library.sh > ../official-images/library/node
+
+      - name: Create PR in official-images
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.MACHINE_USER_PAT }}
+          push-to-fork: ${{ secrets.MACHINE_USER_FORKED_REPOSITORY }}
```

Comment:
Doesn't need to be a secret, just "docker-library/official-images" should be fine. It's a public fork https://github.com/nodejs-github-bot/official-images